### DISCUSSION TITLE
Microsoft.ML.OnnxRuntime.DirectML.1.13.1 has a dev-version in the nuspec file.

### DISCUSSION THREAD

**Discussion by deischi** [microsoft/onnxruntime#13433] [General]
[Microsoft.ML.OnnxRuntime.DirectML.1.13.1](https://github.com/microsoft/onnxruntime/releases/download/v1.13.1/Microsoft.ML.OnnxRuntime.DirectML.1.13.1.zip) is almost a nuget package.

What is it not named .nuget on the release page?
That zip/nuget-file stil has a dev-version (1.13.1-dev-20221022-0833-b353e0b41) in the nuspec file.

Why are there different packages on the release page and on nuget org?

**Comment by faxu** [comment]
This was fixed.
